src/Entity/Category.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+namespace Fei\Service\Filer\Entity;
+
+use Fei\Entity\AbstractEntity;
+
+/**
+ * Class File
+ *
+ * @Entity
+ * @Table(
+ *     name="categories",
+ * )
+ *
+ * @package Fei\Service\Filer\Entity
+ */
+class Category extends AbstractEntity
+{
+    /**
+     * @var int
+     *
+     * @Id
+     * @GeneratedValue(strategy="AUTO")
+     * @Column(type="integer")
+     */
+    protected $id;
+
+    /**
+     * @var string
+     *
+     * @Column(type="string", length=32)
+     */
+    protected $label;
+
+    /**
+     * @var int
+     *
+     * @Column(type="integer")
+     */
+    protected $backend = 1;
```

Comment:
What does "1" mean ? Why no entity here ?

Reply:
no entity yet :)

Merge into master ???? Are you going mad ? :-1: :)